addons/pos_restaurant/static/tests/tours/screen_saver_tour.js

```diff
@@ -0,0 +1,36 @@
+import * as Dialog from "@point_of_sale/../tests/tours/utils/dialog_util";
+import * as Chrome from "@point_of_sale/../tests/tours/utils/chrome_util";
+import { registry } from "@web/core/registry";
+import { PosStore } from "@point_of_sale/app/store/pos_store";
+import { patch } from "@web/core/utils/patch";
+
+patch(PosStore.prototype, {
+    get idleTimeout() {
+        const result = super.idleTimeout;
+        if (result.length > 0) {
+            result[0].timeout = 5000;
+        } // 5 seconds
+        if (result.length > 1) {
+            result[1].timeout = 2000;
+        } // 2 seconds
+        if (result.length > 2) {
+            result[2].timeout = 3000;
+        } // 3 seconds
```

Comment:
change the timeout values to much lower values, just in the order of 10 or 100 ms.